app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -126,6 +128,20 @@ class MembersFragment : BaseFragment() {
                                 adapter = rvAdapter
                                 isRecyclerView = true
                             }
+                        } else {
+                            if (!filterMap["location"].isNullOrEmpty()) {
+
+                                // Here we are checking if location from filter
+                                    // matches with location from any user's profile
+                                val check_location = membersViewModel.userList.any {
+                                    (it.location)?.contains(filterMap["location"]!!, ignoreCase = true) == true
+                                }
+
+                                if (!check_location) {
+                                    Toast.makeText(activity, "Oops! No member found",
```

Comment:
Can you make this a string available in the `strings.xml` resources for future internationalization, please?